rust/src/plugin.rs

```diff
@@ -6,6 +6,17 @@ pub struct Plugin<'a> {
 }
 
 impl<'a> Plugin<'a> {
+    pub unsafe fn from(id: i32, context: &'a Context) -> Plugin<'a> {
```

Comment:
If this is going to be public, could you rename it to `Plugin::from_id` or something more specific?